docs/deployment-process/issue-tracking/azure-devops.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Azure DevOps Work Item Tracking Integration
+description: Configure Azure DevOps work item tracking with Octopus.
+position: 40
+---
+
+**Octopus 2019.7.3** introduced support to integrate Octopus with Azure DevOps work items. The integration includes the ability to:
+- Automatically add links to Azure DevOps work items from releases and deployments in Octopus.
+- Retrieve release notes from Azure DevOps work item comments to assist in automating release note generation.
+
+## How Azure DevOps Integration Works
+
+![Octopus Azure DevOps integration - how it works diagram](images/octo-azure-devops-how-it-works.png "width=500")
+
+1. Associate code changes with their relevant work items in any of the following ways:
```

Comment:
Do you think this warrants a breakout section like the GitHub page did for commits? To clearly explain the different options w/ screenshots?

Reply:
I'd be concerned about disrupting flow: I want to at least lay out the user's set of options in the main flow here.

I wouldn't be against adding a section if further explanation is needed, but I can only imagine an animated screenshot involving "locate, zoom, click, click, click" because these interactions wouldn't convey well in a static screenshot. Maybe something to plan towards?